metals/src/main/scala/scala/meta/internal/metals/BuildTargetInfo.scala

```diff
@@ -218,11 +218,18 @@ class BuildTargetInfo(buildTargets: BuildTargets) {
       .toList
   }
 
-  private def getSources(target: BuildTarget): List[AbsolutePath] = {
+  case class SourceItem(path: AbsolutePath, generated: Boolean) {
```

Comment:
Is `SourceItem` used?

Reply:
Right! deleted https://github.com/scalameta/metals/pull/3720/commits/e380d0d09e6effca0d492352c7024fa5e2995c65